tutorials/tts/FastPitch_MixerTTS_Training.ipynb

```diff
@@ -589,7 +589,7 @@
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "Python 3",
+   "display_name": "Python 3.9.15 ('ptl_venv')",
```

Comment:
nit: we may need to remove these to avoid conflicts every time when we update jupyter notes.